tensorflow/lite/kernels/activations.cc

```diff
@@ -678,8 +685,7 @@ TfLiteStatus PreluPrepare(TfLiteContext* context, TfLiteNode* node) {
 
   output->type = input->type;
 
-  if (output->type == kTfLiteUInt8 || output->type == kTfLiteInt8 ||
-      output->type == kTfLiteInt16) {
+  if (output->type == kTfLiteUInt8 || output->type == kTfLiteInt8) {
```

Comment:
Could you please keep this Int16 support? I think this is added to support some use case.

Reply:
It seems that `PreluEval` only support outputs of type `float`, `uint8_t` and `int8_t`. Do you know in which cases we could have a Prelu with an int16 output?

Reply:
Ah, you are right, the eval function does not handle int16. Thanks for the change.